otter/effect_dispatcher.py

```diff
@@ -32,7 +36,7 @@ def get_simple_dispatcher(reactor):
             Authenticate: perform_authenticate,
             InvalidateToken: perform_invalidate_token,
             Request: perform_request,
-            Retry: perform_retry,
+            Retry: perform_retry
```

Comment:
That trailing comma was intentional (exactly so diffs would be simpler), could you revert this change?